{{cookiecutter.project_dir}}/apps/api/h1st_api/views.py

```diff
@@ -11,7 +11,19 @@
 from django.http import HttpResponse, JsonResponse
 from rest_framework.decorators import api_view
 from rest_framework.parsers import JSONParser
-import h1st_models.translate
+
+# append python path to include ai package
+import sys
+from pathlib import Path
+
+ROOT_DIR = Path(__file__).resolve(strict=True).parent.parent.parent.parent
+sys.path.append(str(ROOT_DIR))
```

Comment:
Can we set the PYTHONPATH in Dockerfile instead? The way we construct `ROOT_DIR` looks awkward to me :D @phamhoangtuan

Reply:
we can do that but @TgithubJ required the code also can run in local without using Dockerfile, so we need to use sys.path.append in the code